Fabric.Authorization.API/RemoteServices/Identity/Providers/IdentityServiceProvider.cs

```diff
@@ -25,6 +25,8 @@ public class IdentityServiceProvider : IIdentityServiceProvider
         private readonly ILogger _logger;
 
         private const string ServiceName = "Fabric.Identity";
+        private const string UsersEndpoint = "users";
+        private const string PrincipalsEndpoint = "principals/";
```

Comment:
should these both contain, or not contain slashes?

Reply:
UsersEndpoint can have a slash for more consistency.